### DISCUSSION TITLE
Everything becomes a visual nightmare as varying parts of the screen are repeated everywhere.

### DISCUSSION THREAD

**Discussion by uBreeze** [ppy/osu#16286] [Q&A] [answered]
I don’t even know how to describe my issue, everything just becomes an absolute clusterfuck the moment I open a song and sometimes just when I open osu!lazer. 

I’m using a Macbook Pro 2011 on High Sierra and the moment I enter a song on osu!lazer, everything just becomes difficult to see. Usually, it only affects the top half of the screen, but sometimes, it does it to the whole screen as well.

The issue remains even after I get off osu!lazer but only starts when I get on it.

[youtube video displaying the issue](https://youtu.be/iWijCTSyC7A)

**Comment by MishterKirby** [comment, marked as answer]
That seems more likely to be a hardware issue. Does it not happen in other programs/running osu! in windowed mode (ALT+Enter)?

**Reply by uBreeze** [reply]
I wouldn’t doubt it’s a hardware issue, my Mac is pretty crap. But this has never happened with any other program. Running osu! in Windowed seems to resolve the issue.

**Reply by peppy** [reply]
If it works in windowed mode I'd recommend using that for now.
